app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -143,6 +143,21 @@ def self.replace_context_secrets(var_name)
         "$$GITHUB_SECRET_#{var_name.upcase}"
       end
 
+      def self.replace_context_runner(var_name)
+        case var_name
+        when "name"
```

Comment:
minor comment: non-interpolating strings should use single quotes (that is what running `rubocop app/` states and I believe it is because it avoids trying to parse and replace stuff in them)